tests/rptest/services/redpanda.py

```diff
@@ -706,7 +706,7 @@ def start(self,
               nodes=None,
               clean_nodes=True,
               start_si=True,
-              parallel: bool = False):
+              parallel: bool = True):
```

Comment:
+1. at one point we had observed that the basic start-up cost of a empty test was like about 10 seconds or so. x500 tests and anything little thing helps!